### ISSUE TITLE
HoverZoom not working on any site

### ISSUE THREAD

**Issue by meldiane** [unarj/hooverZoom#1]
Hi there. I am not sure what to do but the app is not working at all. I have firefox and it was working but when I changed it from 500 to 2000 it just stopped. I deleted and reinstalled but same settings were there. I put everything back to default and still not working. Help?

**Comment by unarj** [comment]
i just set mine for 2000 to test, work as intended once I reloaded the current page (was instant popup with no delay until the reload).  if you're not getting anything at all something weird is going on.  try disabling other addons?

**Comment by meldiane** [comment]
yeah I will go ahead and disable any add ins and get back to you. Thank you
for responding so fast.

**Comment by meldiane** [comment]
no go. I cleared cache and everything to be safe. Any suggestions? I guess
I can try to delete firefox and reinstall.

**Comment by meldiane** [comment]
No go. Guess I am stuck. Did everything possible! Hate to ask but is there
ANYTHING else like you out there?

Here are my settings

[image: Inline image 1]

**Comment by unarj** [comment]
can't see an image, just a tag.

I found my preference file in...
C:\Users\<user>\AppData\Roaming\Mozilla\Firefox\Profiles\<prof>.default\prefs.js

close FF, find the lines that start with...
user_pref("extensions.jid1-ypAaFtWSGV9ANQ@jetpack
...and delete them to rest everything.

for example, I set the load delay to 5 seconds and then deleted this line...
user_pref("extensions.jid1-ypAaFtWSGV9ANQ@jetpack.delay", 50000);
...saved the file, and the default 500 was restored when I started up FF.

good luck!
